test/src/main/java/org/apache/accumulo/test/GetManagerStats.java

```diff
@@ -58,101 +58,103 @@ public static void main(String[] args) throws Exception {
             TableInfo info = status.getValue();
-            out(3, "Table: %s", status.getKey());
-            out(4, "Tablets: %d", info.onlineTablets);
-            out(4, "Records: %d", info.recs);
-            out(4, "Records in Memory: %d", info.recsInMemory);
-            out(4, "Ingest: %.2f", info.ingestRate);
-            out(4, "Queries: %.2f", info.queryRate);
-            out(4, "Major Compacting: %d", info.majors == null ? 0 : info.majors.running);
-            out(4, "Queued for Major Compaction: %d", info.majors == null ? 0 : info.majors.queued);
-            out(4, "Minor Compacting: %d", info.minors == null ? 0 : info.minors.running);
-            out(4, "Queued for Minor Compaction: %d", info.minors == null ? 0 : info.minors.queued);
+            out(3, "Table: %s%n", status.getKey());
+            out(4, "Tablets: %d%n", info.onlineTablets);
+            out(4, "Records: %d%n", info.recs);
+            out(4, "Records in Memory: %d%n", info.recsInMemory);
+            out(4, "Ingest: %.2f%n", info.ingestRate);
+            out(4, "Queries: %.2f%n", info.queryRate);
+            out(4, "Major Compacting: %d%n", info.majors == null ? 0 : info.majors.running);
+            out(4, "Queued for Major Compaction: %d%n",
+                info.majors == null ? 0 : info.majors.queued);
+            out(4, "Minor Compacting: %d%n", info.minors == null ? 0 : info.minors.running);
+            out(4, "Queued for Minor Compaction: %d%n",
+                info.minors == null ? 0 : info.minors.queued);
           }
         }
-        out(2, "Recoveries: %d", server.logSorts.size());
+        out(2, "Recoveries: %d%n", server.logSorts.size());
         for (RecoveryStatus sort : server.logSorts) {
-          out(3, "File: %s", sort.name);
-          out(3, "Progress: %.2f%%", sort.progress * 100);
-          out(3, "Time running: %s", sort.runtime / 1000.);
+          out(3, "File: %s%n", sort.name);
+          out(3, "Progress: %.2f%%%n", sort.progress * 100);
+          out(3, "Time running: %s%n", sort.runtime / 1000.);
         }
-        out(3, "Bulk imports: %s", stats.bulkImports.size());
+        out(3, "Bulk imports: %s%n", stats.bulkImports.size());
         for (BulkImportStatus bulk : stats.bulkImports) {
-          out(4, "Import file: %s", bulk.filename);
-          out(5, "Bulk state %s", bulk.state);
-          out(5, "Bulk start %s", bulk.startTime);
+          out(4, "Import file: %s%n", bulk.filename);
+          out(5, "Bulk state %s%n", bulk.state);
+          out(5, "Bulk start %s%n", bulk.startTime);
         }
 
       }
     }
   }
 
-  private static void out(int indent, String string, Object... args) {
+  private static void out(int indent, String fmtString, Object... args) {
     for (int i = 0; i < indent; i++) {
       System.out.print(" ");
     }
-    System.out.println(String.format(string, args));
+    System.out.printf(fmtString, args);
```

Comment:
I think this makes more sense, so the rest of the file doesn't need to change.

```suggestion
    System.out.printf(fmtString + "%n", args);
```